pkgs/development/python-modules/feedgen/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchurl, python, dateutil, lxml }:
```

Comment:
You seem to be taking python as a function argument in spite of never using it in the expression. This won't affect evaluation (unless python goes missing), but is probably not desirable anyway :)